frontend/src/components/program/DialogActivityCreate.vue

```diff
@@ -252,18 +251,16 @@ export default {
     refreshCopyActivitySource() {
       navigator.permissions.query({ name: 'clipboard-read' }).then(
         (p) => {
-          this.$set(this, 'clipboardPermission', p.state)
-          this.$set(this, 'copyActivitySource', null)
+          this.clipboardPermission = p.state
+          this.copyActivitySource = null
 
-          if (p.state == 'granted') {
+          if (p.state === 'granted') {
             navigator.clipboard
               .readText()
-              .then((url) => {
-                this.getCopyActivitySource(url).then((activityProxy) => {
-                  activityProxy._meta.load.then((activity) =>
-                    this.$set(this, 'copyActivitySource', activity)
-                  )
-                })
+              .then(async (url) => {
+                this.copyActivitySource = await (
```

Comment:
is this better?
i think 2 lines would make this easier to read

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/4823/commits/7fe7f5f83c3ba1befa38bda774505325af4390ad